README.md

```diff
@@ -241,6 +242,24 @@ this command will generate Value Object class into Domain.
 
 ---
 
+#### `infrastructure:make:observer UserObserver User`
+Command for generate a Model Observe class to your project.
+
+**Arguments**
+
+|  Name  |    Description     |
+|:------:|:------------------:|
+|  Name  | Observe name class |
+| Domain |    Domain Name     |
+
+**Options**
+
+|  Name   |          Description          |
+|:-------:|:-----------------------------:|
+| --force | Force create the domain class |
```

Comment:
This for Observer not domain, so please rename it too

Reply:
updated